src/Photos.vue

```diff
@@ -91,7 +91,7 @@ export default {
 	},
 	data() {
 		return {
-			loading: true,
+			loading: false,
```

Comment:
This makes sure the grid is rendered on initial load. It will then load the first batch on its own.